packages/amplication-data-service-generator/scripts/generate-test-data-service.ts

```diff
@@ -34,7 +34,7 @@ async function writeModules(
     modules.map(async (module) => {
       const filePath = path.join(destination, module.path);
       await fs.promises.mkdir(path.dirname(filePath), { recursive: true });
-      await fs.promises.writeFile(filePath, module.code);
+      await fs.promises.writeFile(filePath, STATIC_COMMENT + module.code);
```

Comment:
Undo this change. 
The test files are generated based on the logic on DSG and should not be altered at this point

Reply:
@yuval-hazaz My deepest apologies for overlooking this. I'll be sure to correct the mistake with my latest changes.

Reply:
@yuval-hazaz My deepest apologies for overlooking this. I'll correct the mistake with my latest changes.